test/import/FSHImporter.Profile.test.ts

```diff
@@ -845,6 +899,25 @@ describe('FSHImporter', () => {
         assertCardRule(profile.rules[3], 'component[DiastolicBP]', 2, '*');
         assertFlagRule(profile.rules[4], 'component[DiastolicBP]', true, true, undefined);
       });
+
+      it('should parse contains rule with item declaring a type and flags', () => {
+        const input = `
+        Alias: OffsetExtension = http://hl7.org/fhir/StructureDefinition/observation-timeOffset
+        Profile: ObservationProfile
+        Parent: Observation
+        * component.extension contains OffsetExtension named offset 0..1 MS
+        `;
+
+        const result = importSingleText(input);
+        const profile = result.profiles.get('ObservationProfile');
+        expect(profile.rules).toHaveLength(3);
+        assertContainsRule(profile.rules[0], 'component.extension', {
+          name: 'offset',
+          type: 'http://hl7.org/fhir/StructureDefinition/observation-timeOffset'
+        });
+        assertCardRule(profile.rules[1], 'component.extension[offset]', 0, 1);
+        assertFlagRule(profile.rules[2], 'component.extension[offset]', true, undefined, undefined);
+      });
```

Comment:
Is there any need to add a test to check that the type is correctly imported from an existing FSH defined extension (one that has not been aliased)?

Reply:
Sure, I can do that!

Reply:
Addressed in 65f02b6e679db12f7136d5e68c44c80a4f01ce93